frontend/src/packages/trucks/trucks-api.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiPath, ContentType } from '~/libs/enums/enums.js';
+import { HttpHeader } from '~/libs/packages/http/http.js';
+import { type Http } from '~/libs/packages/http/http.package.js';
+import { type ValueOf } from '~/libs/types/types.js';
+
+import { TruckApiPath } from './libs/enums/enums.js';
+import { type TruckEntity } from './libs/types/types.js';
+
+class TruckApi {
+  private baseUrl: string;
+
+  private http: Http;
```

Comment:
```suggestion
class TruckApi extends HttpApi {
  private baseUrl: string;

  private http: IHttp;
```

Reply:
class TrucksApi

Reply:
When I extend HttpApi, I should include storage in the constructor, as it was in the previous version of this file. You advised me not to use storage, so I'm a bit confused.

Reply:
@denred  I'm sorry, my bad, I forgot that it is used in HttpApi for authentication.